self-hosted-agent/helm/README.md

```diff
@@ -0,0 +1,32 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Would be good to add some free-text description here about what this whole TF stack does, what+who is it for, etc...